.github/scripts/release-verification.sh

```diff
@@ -16,88 +16,173 @@
 # limitations under the License.
 # ------------------------------------------------------------
 
-set -ex
-
-# RELEASE_VERSION_NUMBER is the Radius release version number
-# (e.g. 0.24.0, 0.24.0-rc1)
-RELEASE_VERSION_NUMBER=$1
+set -euo pipefail
+
+# Configuration
+readonly NAMESPACE="radius-system"
+readonly GITHUB_ORG="radius-project"
+readonly GITHUB_REPO="radius"
+
+# Parse arguments: version [os] [arch]
+RELEASE_VERSION_NUMBER="${1:-}"
+OS="${2:-linux}"
+ARCH="${3:-amd64}"
+
+# Cleanup function to remove temporary files and cluster
+cleanup() {
```

Comment:
Do we need cleanup at all? Isn't the VM going to be destroyed after this action runs?